src/fairness/print_hierarchy/print_hierarchy.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+Copyright 2020 Lawrence Livermore National Security, LLC
+(c.f. AUTHORS, NOTICE.LLNS, COPYING)
+
+This file is part of the Flux resource manager framework.
+For details, see https://github.com/flux-framework.
+
+SPDX-License-Identifier: LGPL-3.0
+*/
+#include <iostream>
+#include <sqlite3.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+#include <tuple>
+
+std::string hierarchy = "Bank|User|RawShares\n";
+std::string indent = "";
```

Comment:
I think this should be moved into `main` since it is passed down into `get_sub_banks` as an argument.  I don't think there is a need currently for it to be a global.